DOC-443-bug: add note to fix doc bug (#525)
hazelcast/management-center-docs: docs/modules/deploy-manage/pages/production-checklist.adoc

 - Inconsistencies in the reported metrics may occur if one of the instances goes down for some time.
 - HTTP sessions are not shared between Management Center instances. Load balancing options are limited to sticky sessions or round-robin connections with basic authentication.

> NOTE: If relying on the JSESSIONID cookie for sticky session routing, consider setting the JETTY_WORKER_INSTANCE environment variable distinctly for each Management Center instance to provide a consistent value for configuring your routes appropriately.

Reviewer: ```suggestion
NOTE: If you are relying on the `JSESSIONID` cookie for sticky session routing, consider setting the `JETTY_WORKER_INSTANCE` environment variable distinctly for each Management Center instance to provide a consistent value for configuring your routes.
```
